src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -192,14 +168,40 @@ public double LightElevationDegrees
             set { lightElevationDegrees = value; }
         }
 
-#if DEBUG
-        /// <summary>
-        /// The TestSelectionCommand is used in the WatchSettingsControl
-        /// to test the ability to toggle a boolean effect variable
-        /// representing the selection state.
-        /// </summary>
-        public Dynamo.UI.Commands.DelegateCommand TestSelectionCommand { get; set; }
-#endif
+        private Dictionary<string, Model3D> geometryDictionary= new Dictionary<string, Model3D>();
+
+        public Dictionary<string, Model3D> GeometryDictionary
+        {
+            get
+            {
+                return geometryDictionary;
+            }
+
+            set
+            {
+                geometryDictionary = value;
+            }
+        }
+
+        public List<Model3D> GeometryValues
```

Comment:
@ramramps Please mark these `private` unless they are required to be public for the purposes of data binding. For that matter, we can just make them private members. 

FYI, as an earlier experiment, I tried to mark all of these properties internal, but it seemed to mess with data binding.